dotnet/targets/Xamarin.Shared.Sdk.props

```diff
@@ -9,6 +9,8 @@
 	<Import Project="Xamarin.Shared.Sdk.TargetFrameworkInference.props" />
 
 	<PropertyGroup>
+		<!-- Set to true when using the Microsoft.<platform>.Sdk NuGet. This is used by pre-existing/shared targets to tweak behavior depending on build system -->
+		<UsingAppleNETSdk>true</UsingAppleNETSdk>
```

Comment:
Renaming `_UsingXamarinSdk` to `UsingAppleNETSdk` can be split out into a separate PR.

Reply:
Do we want one of these per platform/SDK type, or is `UsingAppleNETSdk` preferred? Android currently sets `UsingAndroidNETSdk`.  I suppose something more granular could also be added in the future if needed.

Reply:
Moved to https://github.com/xamarin/xamarin-macios/pull/11270.